test/specs/apexLsp.e2e.ts

```diff
@@ -27,25 +27,11 @@ describe('Apex LSP', async () => {
     utilities.log('ApexLsp - Set up the testing environment');
     utilities.log(`ApexLsp - JAVA_HOME: ${EnvironmentSettings.getInstance().javaHome}`);
     testSetup = await TestSetup.setUp(testReqConfig);
+    await utilities.pause(utilities.Duration.seconds(10));
     // Create Apex Class
     await utilities.createApexClassWithTest('ExampleClass');
   });
 
-  step('Verify Extension is Running', async () => {
```

Comment:
We don't actually need this step as we verify for the extension presence during setup